mti_nma/steps/nma/nma.py

```diff
@@ -67,21 +68,36 @@ def run(self, **kwargs):
         nma_data_dir.mkdir(parents=True, exist_ok=True)
 
         # run NMA on avg mesh and save results to local stagings
-        w, v, wfig = run_nma(polydata)
+        w, v = run_nma(polydata)
+        draw_whist(w)
+        vmags = get_eigvec_mags(v)
+
+        fig_path = nma_data_dir / Path('w_fig.pdf')
+        plt.savefig(fig_path, format='pdf')
         w_path = nma_data_dir / Path('eigvals.npy')
         np.save(w_path, w)
         v_path = nma_data_dir / Path('eigvecs.npy')
         np.save(v_path, v)
-        fig_path = nma_data_dir / Path('w_fig.pdf')
-        plt.savefig(fig_path, format='pdf')
+        vmags_path = nma_data_dir / Path('eigvecs_mags.npy')
+        np.save(vmags_path, vmags)
 
         # Create manifest with eigenvectors, eigenvalues, and hist of eigenvalues
         self.manifest = pd.DataFrame({
-            'Label': 'nma_avg_nuc_mesh',
+            "Label": "nma_avg_nuc_mesh",
```

Comment:
Not a part of this PR but any extra metadata you want to directly expose to objects in Quilt can be set during the init as well. With `metadata_columns` similar to `filepath_columns`. Really optional but useful in the case where you are saving a bunch of images and have an FOVId or such.